extra/install_dep.sh

```diff
@@ -28,8 +28,8 @@ if [ y`uname`y = yLinuxy ]; then
 		$SUDO apt update
 		
 		# for generic
-		$SUDO apt install -y cmake make gcc libnl-3-dev libglib2.0-dev zlib1g kmod
-		$SUDO apt install -y libnl-3-dev libglib2.0-0 libkmod-dev libgoogle-perftools-dev
+		$SUDO apt install -y cmake make gcc zlib1g kmod
+		$SUDO apt install -y libnl-3-dev libnl-genl-3-dev libglib2.0-0 libkmod-dev libgoogle-perftools-dev
```

Comment:
Do you want to add this so it matches the CMakeList?

I had thought libnl-genl-3-dev was brought in automatically when you install libnl-3-dev.

Reply:
`libnl-genl-3-dev` and `libnl-3-dev` are two different package in Debian, installing `libnl-3-dev` will NOT install `libnl-genl-3-dev` automatically.

I got a cmake error "LIBNL_GENL_LIB not found" after I run `install_dep.sh`, so I add this package to this script.